std/http/server_test.ts

```diff
@@ -392,13 +396,13 @@ test({
 
 test({
   name: "serveTLS",
-  // FIXME(bartlomieju): hangs on windows, cause can't do `Deno.kill`
-  ignore: true,
+  ignore: false,
```

Comment:
ignore is false by default. I think it would be better if you remove `ignore: false`.

Reply:
right, thanks @kerolloz 👍